docs/container.md

```diff
@@ -0,0 +1,102 @@
+<!--
+SPDX-FileCopyrightText: Red Hat, Inc.
+SPDX-License-Identifier: GPL-2.0-or-later
+-->
+
+# Using the ovirt-imageio container
+
+## Overview
+
+You can use ovirt-imageio to transfer images without oVirt using the
+ovirt-imageio container image.
+
+This way, you can run an image container for every transfer,
+exposing the container for remote access, and connecting the
+relevant storage. When the transfer is done, the user will
+unexpose the container and terminate it.
+
+In this mode the imageio server will serve one ticket, injected into the
+container when starting, instead of adding a ticket using the control
+service. Actually, the control service is disabled.
+
+To obtain the public container image you can run:
+
+    podman pull quay.io/ovirt/ovirt-imageio
+
+## Run the container
+
+To run the container and serve an image, you need to mount the
+image in the container, and load a ticket to expose it.
+Note that the path for the file in the ticket needs to match the
+path where we mounted the images folder in the container.
+
+For example, if we mount a `disk.raw` image in `/var/tmp`, the ticket
+should point to the file in that directory:
+
+    "url": "file:///var/tmp/disk.raw"
+
+Alternatively, we can mount the nbd socket path in the container
+and request the image with a nbd ticket:
+
+    "url": "nbd:unix:/tmp/nbd.sock"
```

Comment:
This is possible but requires running qemu-nbd on the host, which requires installing
additional packages and knowing how to run qemu-nbd. I think this should be done inside
the imageio-container to simplify the usage. So I would not document there this option.